packages/loader/driver-utils/src/blobCacheStorageService.ts

```diff
@@ -23,6 +23,6 @@ export class BlobCacheStorageService extends DocumentStorageServiceProxy {
             return blob;
         }
 
-        return this.internalStorageService.read(id);
+        return this.internalStorageService.readString(id);
```

Comment:
Similar comment to one in ChannelStorageService  - read() can't return unmodified result of readString().
Plus this class is mssing readBlob() & readString() APIs.